python/testmonitor/results/README.md

```diff
@@ -0,0 +1,20 @@
+# SystemLink Test Monitor results example
+
+This is an example of uploading test results to the SystemLink Test Monitor service.
```

Comment:
Is this description new, or is it taken from the C# one? If it's new, should we ask Jenna to review it?

Reply:
This is new.  I'll add Jenna to the review.